app/upgrades.go

```diff
@@ -35,13 +35,12 @@ func (app UmeeApp) RegisterUpgradeHandlers(bool) {
 	app.registerUpgrade3_3to4_0(upgradeInfo)
 	app.registerUpgrade4_0_1(upgradeInfo)
 	app.registerUpgrade4_1(upgradeInfo)
-	app.registerUpgrade4_1_0rc3(upgradeInfo)
 	app.registerUpgrade4_2(upgradeInfo)
 }
 
 // performs upgrade from v4.1 to v4.2
 func (app *UmeeApp) registerUpgrade4_2(upgradeInfo upgradetypes.Plan) {
-	const planName = "v4.2"
+	const planName = "v4.2-beta1"
```

Comment:
@gsk967 for the IBC-quota testnet, the upgrade name has to match this string.

Later we will rename it as described in the PR details.